app/lib/ui/icons/generated_svg_icons.dart

```diff
@@ -0,0 +1,177 @@
+  ),
+  CompiledSvgIcon.trash2: const _InlineBytesLoader(
+    'Yi2IAAEpAADAQQAAwEEdAAAA/wEBAwAAgEAAAOA/AAD//xsAAAACAAAAAAEEAAAAAAAgQQAAMEEAACBBAACIQRsAAQACAAAAAAEEAAAAAAAAAGBBAAAwQQAAYEEAAIhBGwACAAYAAAAAAQIBAgEUAAAAAAAAAJhBAADAQAAAmEEAAKBBAACYQSnWqEEp1pBBAACwQQAAiEEAALBBAADgQAAAsEFep7xAAACwQQAAoEAp1qhBAACgQAAAoEEAAKBAAADAQBsAAwACAAAAAAEEAAAAAAAAAEBAAADAQAAAqEEAAMBAGwAEAAYAAAAAAQIBAgEUAAAAAAAAAABBAADAQAAAAEEAAIBAAAAAQbxOOUCvUw5BAAAAQAAAIEEAAABAAABgQQAAAEBRrHFBAAAAQAAAgEG8TjlAAACAQQAAgEAAAIBBAADAQDAeAAAAAP//HgEAAAD//x4CAAAA//8eAwAAAP//HgQAAAD//w==',
+  ),
+  CompiledSvgIcon.trash: const _InlineBytesLoader(
+    'Yi2IAAEpAADAQQAAwEEdAAAA/wEBAwAAgEAAAOA/AAD//xsAAAAGAAAAAAECAQIBFAAAAAAAmEEAAMBAAACYQQAAoEEAAJhBKdaoQSnWkEEAALBBAACIQQAAsEEAAOBAAACwQV6nvEAAALBBAACgQCnWqEEAAKBAAACgQQAAoEAAAMBAGwABAAIAAAAAAQQAAAAAAAAAQEAAAMBAAACoQQAAwEAbAAIABgAAAAABAgECARQAAAAAAAAAAEEAAMBAAAAAQQAAgEAAAABBvE45QK9TDkEAAABAAAAgQQAAAEAAAGBBAAAAQFGscUEAAABAAACAQbxOOUAAAIBBAACAQAAAgEEAAMBAMB4AAAAA//8eAQAAAP//HgIAAAD//w==',
+  ),
+  CompiledSvgIcon.upload: const _InlineBytesLoader(
+    'Yi2IAAEpAADAQQAAwEEdAAAA/wEBAwAAgEAAAOA/AAD//xsAAAACAAAAAAEEAAAAAABAQQAAQEAAAEBBAABwQRsAAQADAAAAAAEBBgAAAAAAAIhBAAAAQQAAQEEAAEBAAADgQAAAAEEbAAIABgAAAAABAgECARQAAAAAAAAAqEEAAHBBAACoQQAAmEEAAKhBKdagQSnWoEEAAKhBAACYQQAAqEEAAKBAAACoQbxOeUAAAKhBAABAQCnWoEEAAEBAAACYQQAAQEAAAHBBMB4AAAAA//8eAQAAAP//HgIAAAD//w==',
+  ),
+  CompiledSvgIcon.user: const _InlineBytesLoader(
+    'Yi2IAAEpAADAQQAAwEEdAAAA/wEBAwAAgEAAAOA/AAD//xsAAAAGAAAAAAECAQIBFAAAAAAAmEEAAKhBAACYQQAAmEEAAJhBr1OGQVGsiUEAAHBBAABwQQAAcEEAABBBAABwQbxO2UAAAHBBAACgQK9ThkEAAKBAAACYQQAAoEAAAKhBGwABAAYAAAAAAgICAgMaAAAAAAAAAEBBAABAQJNSY0EAAEBAAACAQdlamUAAAIBBAADgQAAAgEGTUhNBk1JjQQAAMEEAAEBBAAAwQW2tHEEAADBBAAAAQZNSE0EAAABBAADgQAAAAEHZWplAba0cQQAAQEAAAEBBAABAQDAeAAAAAP//HgEAAAD//w==',
+  ),
+  CompiledSvgIcon.x: const _InlineBytesLoader(
+    'Yi2IAAEpAADAQQAAwEEdAAAA/wEBAwAAgEAAAOA/AAD//xsAAAACAAAAAAEEAAAAAACQQQAAwEAAAMBAAACQQRsAAQACAAAAAAEEAAAAAAAAAMBAAADAQAAAkEEAAJBBMB4AAAAA//8eAQAAAP//',
+  ),
+};
+
+BytesLoader compiledSvgLoader(CompiledSvgIcon icon) =>
+    _compiledSvgLoaders[icon]!;
+
+Widget compiledSvgWidget(
+  CompiledSvgIcon icon, {
+  double? size,
+  Color? color,
+  BoxFit fit = BoxFit.contain,
+  Alignment alignment = Alignment.center,
+}) {
+  return VectorGraphic(
+    loader: compiledSvgLoader(icon),
+    width: size,
+    height: size,
+    fit: fit,
+    alignment: alignment,
+    colorFilter: color != null
+        ? ColorFilter.mode(color, BlendMode.srcIn)
+        : null,
+  );
+}
+
+class _InlineBytesLoader extends BytesLoader {
+  const _InlineBytesLoader(this.base64Data);
+  final String base64Data;
+  @override
+  Future<ByteData> loadBytes(BuildContext? context) async =>
```

Comment:
This function most likely should not be async. Looking at the goldens, it seems that it does not complete immediately. Most likely `SynchronousFuture` should be used here.